.github/workflows/cleanup.yml

```diff
@@ -38,6 +38,8 @@ on:
         required: false
 
 env:
+  IAM_ERROR_FILENAME: '/tmp/iam_error.txt'
```

Comment:
Instead of /tmp I might use runner.temp 

https://docs.github.com/en/actions/learn-github-actions/contexts#runner-context